src/storage/src/row_serde/value_serde.rs

```diff
@@ -140,6 +141,21 @@ impl ValueRowSerde for ColumnAwareSerde {
     }
 }
 
+#[derive(Clone)]
+pub struct PartialRowDeserializer(Deserializer);
+
+impl PartialRowDeserializer {
+    pub fn new(inner: Deserializer) -> Self {
+        Self(inner)
+    }
+}
+
+impl ValueRowDeserializer for PartialRowDeserializer {
+    fn deserialize(&self, encoded_bytes: &[u8]) -> Result<Vec<Datum>> {
+        self.0.deserialize(encoded_bytes)
+    }
+}
+
```

Comment:
This looks to behave the exactly same as `Deserializer`. 🤔 How could it be partial (or the original one not be partial)?

Reply:
yes I should get rid of this dummy type.